app.js

```diff
@@ -65,6 +65,7 @@ if (!config.useSSL && config.protocolUseSSL) {
 
 // logger
 app.use(morgan('combined', {
+  skip: (req, _) => req.headers['user-agent'] === 'hedgedoc-container-healthcheck/1.0' && req.ip === '127.0.0.1',
```

Comment:
Please extract the check into a separate function. Like `IsRequestFromContainerHealthCheck()` or something.

Reply:
OK, I'll fix it.